vm/build.rs

```diff
@@ -49,6 +49,11 @@ fn main() {
             println!("cargo:rustc-cfg=ossl111");
         }
     }
+    if let Ok(v) = env::var("DEP_OPENSSL_CONF") {
```

Comment:
updating build document for this?

Reply:
No, it's set by the openssl-sys crate: https://doc.rust-lang.org/cargo/reference/build-script-examples.html#using-another-sys-crate